CDCgov/prime-reportstream
# Make facilities table load async from the report details

This PR wraps the facilities table in a `Suspense` so it does not hold up the visibility of already loaded data in the UI.

Test Steps:
1. pull `kevinhaube/2785-async-fetch
2. `yarn` and `yarn start`
3. navigate to the details page of a report and observe the behavior

## Changes
![Screenshot from 2021-10-28 15-10-24](https://user-images.githubusercontent.com/4022304/139320308-562a651f-920f-4fb1-ad7f-8a3b8010237a.png)

## Checklist

### Testing

### Process

## Fixes
- #2785 

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints? 
No
- Does this PR include changes in authentication and/or authorization of existing endpoints?
No
- Does this change introduce new dependencies that need vetting?
No
- Does this change require changes to our infrastructure?
No
- Does logging contain sensitive data?
No
- Does this PR include or remove any sensitive information itself?
No

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
